serve-web/src/Migrations/Version20210623150012.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20210623150012 extends AbstractMigration
+{
+    public function getDescription() : string
```

Comment:
Don't mean to be a pain the `:` after the parenthesis have spaces but in other files they don't. 

Looking at Digideps which has the formatting it looks like these spaces shouldn't be here

Reply:
Actioned ✅ 

As per Slack - I'll add some pre-commit config to the project on a new PR and run PHP CS fixer on all the files.